src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/SubtypeTest.java

```diff
@@ -10,50 +10,88 @@
 import uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType;
 
 import static java.util.Arrays.asList;
-import static org.assertj.core.api.Assertions.assertThat;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.EnvelopeCreator.inputScannableItem;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType.CHERISHED;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType.COVERSHEET;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType.FORM;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType.OTHER;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType.SSCS1;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType.WILL;
 
+@SuppressWarnings("checkstyle:LineLength")
 public class SubtypeTest {
 
+    public static String SOME_SUBTYPE = "foo";
+
     @Rule
     public final JUnitSoftAssertions softly = new JUnitSoftAssertions();
 
     @Test
     public void should_map_scannable_item_document_types_correctly() {
         asList(
-            new TestCase(inputScannableItem(InputDocumentType.SSCS1), DocumentType.OTHER, DocumentSubtype.SSCS1),
-            new TestCase(inputScannableItem(InputDocumentType.WILL), DocumentType.OTHER, DocumentSubtype.WILL),
-            new TestCase(inputScannableItem(InputDocumentType.COVERSHEET),
-                DocumentType.OTHER,
-                DocumentSubtype.COVERSHEET
-            ),
-            new TestCase(inputScannableItem(InputDocumentType.CHERISHED),
-                DocumentType.CHERISHED, null),
-            new TestCase(inputScannableItem(InputDocumentType.OTHER), DocumentType.OTHER, null)
-        ).forEach(testCase -> {
+            // null subtype
+            new TestCase(new Given(SSCS1, null), new Then(DocumentType.OTHER, DocumentSubtype.SSCS1)),
+            new TestCase(new Given(WILL, null), new Then(DocumentType.OTHER, DocumentSubtype.WILL)),
+            new TestCase(new Given(COVERSHEET, null), new Then(DocumentType.COVERSHEET, null)),
+            new TestCase(new Given(CHERISHED, null), new Then(DocumentType.CHERISHED, null)),
+            new TestCase(new Given(OTHER, null), new Then(DocumentType.OTHER, null)),
+            new TestCase(new Given(FORM, null), new Then(DocumentType.FORM, null)),
+            // non-null subtype
+            new TestCase(new Given(SSCS1, SOME_SUBTYPE), new Then(DocumentType.OTHER, DocumentSubtype.SSCS1)),
```

Comment:
Maybe it's worth renaming `Given` and `Then` classes to something like `Input` and `ExpectedOutput` and create `givenInput` and `expectedOutput` helper methods. Then the class names would be clearer about what they represent and these lines would looks like this:
```
new TestCase(givenInput(CHERISHED, null), expectedOutput(DocumentType.CHERISHED, null))
```

Reply:
Also, maybe instead of `TestCase` class we could have a method doing the assertion?